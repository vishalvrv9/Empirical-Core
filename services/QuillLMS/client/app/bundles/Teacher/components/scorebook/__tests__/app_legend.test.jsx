import React from 'react';
import { shallow } from 'enzyme';

import AppLegend from '../app_legend.jsx'

describe('AppLegend component', () => {

  it('should render', () => {
    const wrapper = shallow(<AppLegend/>)
    expect(wrapper).toMatchSnapshot();
  });

})
