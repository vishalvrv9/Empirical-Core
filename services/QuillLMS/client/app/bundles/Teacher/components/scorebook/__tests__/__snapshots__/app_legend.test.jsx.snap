// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLegend component should render 1`] = `
<div
  className="icons-wrapper icon-legend app-legend"
>
  <div
    className="icons"
  >
    <div
      className="icon"
      onClick={[Function]}
    >
      <div
        className="icon-wrapper icon-diagnostic-embossed"
      />
      <div
        className="icons-description-wrapper"
      >
        <p
          className="title"
        >
          Quill Diagnostic
        </p>
        <p
          className="description"
        >
          Identify Learning Gaps
        </p>
      </div>
    </div>
    <div
      className="icon"
      onClick={[Function]}
    >
      <div
        className="icon-wrapper icon-lessons-embossed"
      />
      <div
        className="icons-description-wrapper"
      >
        <p
          className="title"
        >
          Quill Lessons
        </p>
        <p
          className="description"
        >
          Shared Group Lessons
        </p>
      </div>
    </div>
    <div
      className="icon"
      onClick={[Function]}
    >
      <div
        className="icon-wrapper icon-connect-embossed"
      />
      <div
        className="icons-description-wrapper"
      >
        <p
          className="title"
        >
          Quill Connect
        </p>
        <p
          className="description"
        >
          Combine Sentences
        </p>
      </div>
    </div>
    <div
      className="icon"
      onClick={[Function]}
    >
      <div
        className="icon-wrapper icon-flag-embossed"
      />
      <div
        className="icons-description-wrapper"
      >
        <p
          className="title"
        >
          Quill Proofreader
        </p>
        <p
          className="description"
        >
          Practice Mechanics
        </p>
      </div>
    </div>
    <div
      className="icon"
      onClick={[Function]}
    >
      <div
        className="icon-wrapper icon-puzzle-embossed"
      />
      <div
        className="icons-description-wrapper"
      >
        <p
          className="title"
        >
          Quill Grammar
        </p>
        <p
          className="description"
        >
          Fix Errors In Passages
        </p>
      </div>
    </div>
  </div>
</div>
`;
